packages/fleather/lib/src/widgets/editor.dart

```diff
@@ -81,11 +81,13 @@ typedef FleatherEmbedBuilder = Widget Function(
 /// Only supports "horizontal rule" embeds.
 Widget defaultFleatherEmbedBuilder(BuildContext context, EmbedNode node) {
   if (node.value.type == 'hr') {
-    final theme = FleatherTheme.of(context)!;
+    final fleatherThemeData = FleatherTheme.of(context)!;
+
     return Divider(
-      height: theme.paragraph.style.fontSize! * theme.paragraph.style.height!,
-      thickness: 2,
-      color: Colors.grey.shade200,
+      height: fleatherThemeData.paragraph.style.fontSize! *
```

Comment:
Now it makes sense to read the height from the `HorizontalRuleTheme` too with `fleatherThemeData.paragraph.style.fontSize! * fleatherThemeData.paragraph.style.height!` as the default value. What do you think?

Reply:
Done